docs/src/css/components/_dialog.scss

```
@@ -7,6 +7,16 @@ dialog {
   box-shadow: var(--ix-shadow);
   max-width: 28rem;
 
+  position: fixed; 
```

Stylelint (17.11.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)